modules/EvseV2G/v2g.hpp

```diff
@@ -166,6 +166,8 @@ typedef struct keylogDebugCtx {
     bool inClientRandom;
     bool inMasterSecret;
     uint8_t hexdumpLinesToProcess;
+    int udp_socket;
+    std::string udp_buffer{};
```

Comment:
the `{}` in case of string is not really necessary

Reply:
True